marshmallow/schema.py

```diff
@@ -665,7 +670,6 @@ def _deserialize(
                 fields = {
                     field_obj.data_key or field_name
                     for field_name, field_obj in fields_dict.items()
-                    if not field_obj.dump_only
```

Comment:
This would change the behavior of the `if unknown == INCLUDE:` branch which is supposed to keep all values.

Reply:
See #875.

Reply:
Even if I kept that line, the error is already stored above; I'd have to check above for `INCLUDE` before storing the error. Am I wrong? Also see my comment below with my opinion on `INCLUDE` for `dump_only`